GoRogue/MapGeneration/Region.cs

```diff
@@ -332,15 +325,15 @@ public virtual Region Rotate(double degrees, Point origin)
         /// Adds a sub-region to this region.
         /// </summary>
         /// <param name="region">The region you wish to add as a sub-region</param>
-        public void Add(Region region)
+        public void AddRegion(Region region)
         {
             _subRegions.Add(region);
         }
         /// <summary>
         /// Removes a sub-region whose name matches with the string given.
         /// </summary>
         /// <param name="name">Key of sub-region to remove.</param>
-        public void Remove(string name)
+        public void RemoveRegion(string name)
```

Comment:
See above; would `RemoveSubRegion` be better?

Reply:
Agreed